lib/Runtime/Library/TypedArray.h

```diff
@@ -217,6 +217,43 @@ namespace Js
     };
 
     template <typename TypeName, bool clamped = false, bool virtualAllocated = false>
+    class TypedArray;
+
+    // These are referenced in the TypedArray implementation, so we need to forward-typedef these.
+
+#if defined(__clang__)
+// hack for clang message: "...add an explicit instantiation declaration to .."
+#define __EXPLICIT_INSTANTINATE_TA(x) x;
```

Comment:
Clang needs an additional semicolon here? e.g.
`typedef TypedArray<int8> Int8Array;;`

That's weird.

Reply:
Or is that just because the next set has multiple statements, and the semicolon is required there but not here?

Reply:
That's it. I'll remove the extra semicolon, simplify this whole block, and reupload.